parent/pom.xml

```diff
@@ -38,7 +38,7 @@
     <version.agrona>1.4.0</version.agrona>
     <version.animal-sniffer>1.18</version.animal-sniffer>
     <version.assertj>3.15.0</version.assertj>
-    <version.atomix>3.2.0-SNAPSHOT</version.atomix>
+    <version.atomix>0.23.0-SNAPSHOT</version.atomix>
```

Comment:
We should add atomix in our Zeebe BOM since it's now part of our project, and then we don't need to specify a version here.